docroot/modules/custom/yukon_migrate/migrations/nodes/yukon_migrate_documents_page_translations.yml

```diff
@@ -0,0 +1,95 @@
+# Migrates all revisions and all revision translations.
+id: yukon_migrate_documents_page_translations
+label: Yukon Migrate Translation for Documents Nodes
+audit: true
+migration_tags:
+  - Drupal 7
+  - node
+  - Basic Pages
+migration_group: legacy
+source:
+  plugin: d7_node_entity_translation
+  node_type: [documents, documents_non_branded]
+  constants:
+    default_text_format: full_html
+process:
+  nid:
+    plugin: migration_lookup
+    migration: yukon_migrate_documents_page
+    source: entity_id
+    no_stub: true
+
+  title: title_field
+  # Todo: Replace this part with migration lookup after we migrate users.
+  uid: uid
+  status: status
+  created: created
+  changed: changed
+  promote: promote
+  sticky: sticky
+  revision_uid: revision_uid
+  revision_log: log
+  revision_timestamp: timestamp
+  content_translation_source: source
+  content_translation_outdated: translate
+  langcode: language
+
+  field_page_description: field_description
+
+  body/value: field_document_description/0/value
+  body/format: constants/default_text_format
+
+  field_document_upload:
+    plugin: migration_lookup
+    source: field_document_upload/0/fid
+    migration: yukon_migrate_document
+    no_stub: true
+
+  field_last_updated:
+    plugin: format_date
+    from_format: 'Y-m-d H:i:s'
+    to_format: 'Y-m-d\TH:i:s'
+    source: field_last_updated/0/value
+
+  field_pages: field_pages
+
+  field_category:
+    - plugin: migration_lookup
+      source: field_category
+      migration: yukon_migrate_category
+      no_stub: true
+
+  field_department_term:
+    plugin: migration_lookup
+    migration: yukon_migrate_department
+    source: field_department_term
+    no_stub: true
+
+  field_contact/value: field_contact/0/value
+  field_contact/format: constants/default_text_format
+
+  path/alias: alias
+  path/pathauto: pathauto
+
+  moderation_state: moderation_state
+  field_social_sharing: field_social_sharing
+
+  field_yukon_editorial_team:
+    - plugin: migration_lookup
+      source: field_yukon_editorial_team
+      migration: yukon_migrate_teams
+      no_stub: true
+
+  field_feedback_form: field_feedback_form
+  field_document_type: field_document_type
```

Comment:
Don't we need to add migration for the field_documents_non_branded field somewhere here?